docs/source/models/gbd2017/causes/neonatal_other/index.rst

```diff
@@ -36,3 +36,38 @@ neonatal disorders.
 
 .. todo::
    Describe cause model
+
+Restrictions
+------------
```

Comment:
We have yet to finalize a document structure, but I've been putting Restrictions as a subsection within Model Assumptions and Limitations, which I've been putting after the Data Description section. I'm open to discussion though.

Since you don't have headings for the other sections yet, it doesn't matter too much at this stage, but eventually we should standardize. I'll create a pull request for the cause model template soon so we can work out details there.

Reply:
Thanks for creating the additional page! I will update this page with those if we decide on a format before closing this PR.